packages/react/src/JsonForms.tsx

```diff
@@ -77,6 +77,17 @@ export class JsonFormsDispatchRenderer extends React.Component<
     }
   }
 
+  componentDidUpdate(prevProps: JsonFormsProps) {
+    if (prevProps.schema != this.props.schema) {
```

Comment:
```suggestion
    if (prevProps.schema !== this.props.schema) {
```
Always make sure to use strict inequality (and equality) checks to avoid unexpected bugs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_inequality

Reply:
Thank you, I must have missed that here!